### PR TITLE
General geography category :D

### PR DESCRIPTION
**Pull request by Kreusada**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [x] New feature

### Description of the changes

Added a general geography category into the core trivia categories. Has a range of easy to challenging questions, and the yaml is validated. My Discord user is Kreusada#0518 if you'd be needing this.

Thanks!

### CODE REVIEW

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+- McMurdo Dry Valleys
+- McMurdo Valleys
+In what country can you visit Machu Picchu?:
+- Peru
+Which African nation has the most pyramids?:
+- Sudan
+What African country served as the setting for Tatooine in Star Wars?:
+- Tunisia
+What is the oldest city in the world?:
+- Damascus
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml]: I'm not sure where you're sourcing this from, but I find conflicting answers.
<https://www.theguardian.com/cities/2015/feb/16/whats-the-oldest-city-in-the-world>

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+- Mount Everest
+- Everest
+What is the capital of Australia?:
+- Canberra
+Riyadh is the capital of what Middle-Eastern country?:
+- Saudi Arabia
+What is the smallest independent country on Earth?:
+- Vatican City
+What is the deepest point in Earth's oceans?:
+- Mariana Trench
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml:54]: The area is called Mariana Trench, but the point itself is called [Challenger Deep](https://en.wikipedia.org/wiki/Challenger_Deep). Perhaps add both as options?

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+What mountain is closest to the Moon?:
+- Mount Chimborazo
+- Chimborazo
+What continent contains the most fresh water?:
+- Antarctica
+What ocean is home to 75% of the Earth's volcanoes?:
+- Pacific 
+- Pacific Ocean
+What is the largest city in the world based on surface area?:
+- Hulunbuir
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml]: Not sure where you are sourcing this, but this seems to be different based on your definition of a "city".
<https://en.wikipedia.org/wiki/List_of_largest_cities_by_area>

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+What is the only major city located on two continents?:
+- Istanbul
+What is the coldest sea on Earth?:
+- White Sea
+- White
+What is America's largest city by surface area?:
+- Yakutat
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml]: This is listed as a [borough](https://en.wikipedia.org/wiki/Yakutat,_Alaska), not a city. The largest city is supposedly [Sitka](https://en.wikipedia.org/wiki/List_of_United_States_cities_by_area).

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+What is the most populated country in Africa?:
+- Nigeria
+What Asian country has Kuala Lumpur as its capital?:
+- Malaysia
+What European country is home to Transylvania?:
+- Romania
+What body of water separates Africa and Europe?:
+- Strait of Gibraltar
+- Gibraltar
+Which of the following continents is intersected by the Equator?:
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml]: "Which of the following" with nothing following it.

File: redbot/cogs/trivia/data/lists/geography.yaml [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+What is the state capital of Oklahoma?:
+- Oklahoma City
+What is the oldest city in Texas?:
+- Nacogdoches
+What is the flattest U.S. state?:
+- Florida
+What country borders the Caspian Sea, Persian Gulf, and Gulf of Oman?:
+- Iran
+Which U.S. state has the most miles of rivers?:
+- Nebraska
```
Comment (Flame442) [inline redbot/cogs/trivia/data/lists/geography.yaml]: What are your sources for this? I found conflicting answers.
<https://www.farmprogress.com/husker-harvest-days/no-nebraska-doesnt-have-most-miles-river>

**Review by Flame442 — CHANGES_REQUESTED** [review]
A few things to handle, but otherwise looking good!

**Comment by Kreusada** [comment]
Thankyou for picking up on these errors. I was able to correct them all so now it should be looking good.

**Review by Flame442 — APPROVED** [review]
All set, thanks for your contribution!

**Pull request merged**